### PR TITLE
Marketplace services - Add  filter/transformer tags

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/36557

## Description
- Added `TRANSFORMER` and `FILTER` tags to the class `PackTags`.
- Added `_is_transformer` and `_is_filter` properties to the class `Pack`.
- Update `_collect_pack_tags` and `collect_content_items` methods in order to add the filter/transformer tags to the pack if a the pack contains a filter/transformer script.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Tests/Marketplace/marketplace_services.py [hunk 1]
```diff
@@ -1596,6 +1604,8 @@ def _collect_pack_tags(self, user_metadata, landing_page_sections, trending_pack
         tags |= self._get_tags_from_landing_page(landing_page_sections)
         tags |= {PackTags.TIM} if self._is_feed else set()
         tags |= {PackTags.USE_CASE} if self._use_cases else set()
+        tags |= {PackTags.TRANSFORMER} if self._is_transformer else set()
+        tags |= {PackTags.FILTER} if self._is_filter else set()
```
Comment (guyfreund) [inline Tests/Marketplace/marketplace_services.py:1608]:
```suggestion
        tags |= {PackTags.TRANSFORMER} if self._contains_transformer else set()
        tags |= {PackTags.FILTER} if self._contains_filter else set()
```

**Review by guyfreund — CHANGES_REQUESTED** [review]
Looks good, just one main note 💪

**Comment by adi88d** [comment]
@guyfreund I added the properties `contains_filter` and `contains_transformer` to the Pack class

**Review by guyfreund — CHANGES_REQUESTED** [review]
Please run testing-upload and attach a successful run (make sure you see the new tag for new/modified and existing packs that contains transformer/filter and the tags for both)

File: Tests/Marketplace/marketplace_services.py [hunk 2]
```diff
@@ -307,6 +309,18 @@ def uploaded_integration_images(self):
...
+        """ bool: whether the pack contains a transformer script.
+        """
+        return self._contains_transformer
+
+    @property
+    def contains_filter(self):
+        """ bool: whether the pack contains a filter script.
+        """
+        return self._contains_filter
+
```
Comment (guyfreund) [inline Tests/Marketplace/marketplace_services.py:323]:
```suggestion
```
no need

**Review by guyfreund — CHANGES_REQUESTED** [review]

**Comment by adi88d** [comment]
@guyfreund you can see the upload flow [here](https://app.circleci.com/pipelines/github/demisto/content/97777/workflows/2965af11-2c95-40f2-a099-0c93df3558a7/jobs/391893)

**Comment by guyfreund** [comment]
> @guyfreund you can see the upload flow [here](https://app.circleci.com/pipelines/github/demisto/content/97777/workflows/2965af11-2c95-40f2-a099-0c93df3558a7/jobs/391893)

your upload is failing + you still have the getters in the pr

**Review by yaakovi — APPROVED** [review]
update the link to the build

**Comment by adi88d** [comment]
testing upload flow link is [here](https://app.circleci.com/pipelines/github/demisto/content/98034/workflows/7f89cf65-91ba-4877-a7ec-92a89af7bfbb)

**Review by guyfreund — APPROVED** [review]
NICE
image

**Pull request merged**